testsuite/functional-testsuite/src/test/ft-resources/testsuite-oss/framework/mode_button.t.js

```diff
@@ -13,9 +13,9 @@
 /**
  * Tests mode buttons are present and are working.
  */
-StartTest(function (t) {
+StartTest(function(t) {
 
-  var checkMode = function (mode, collapsed, next) {
+  var checkMode = function(mode, collapsed, next) {
     var menu = t.cq1('nx-feature-menu');
```

Comment:
The usage of `t` here results in `t.diag(...)` messages showing up at the wrong level. i.e. they will always show up at the top level instead of nested to the level where `checkmode` is called. Passing `t` into this function would result `t` being captured at the level/state.